src/widgets/search-box/search-box.js

```diff
@@ -309,31 +273,6 @@ export default function searchBox({
 // 'onpropertychange' is the IE8 alternative until we support IE8
```

Comment:
this code can be removed, IMO we should have used the React version for consistency, but understand that this is less work.